pyatv/const.py

```diff
@@ -286,3 +286,19 @@ class FeatureName(Enum):
 
     TurnOff = 34
     """Turn off device."""
+
+
+class ProtocolState(Enum):
```

Comment:
This file (`const.py`) is only for things in the public interface, so you should move this back to `protocol.py` as the protocol state is only internal.